src/lib.rs

```diff
@@ -851,6 +851,25 @@ pub type Ixs = isize;
 /// </tr>
 /// </table>
 ///
```

Comment:
Can we put a heading here, so that it's possible to find this?

I also want to disclaim the `as_slice` methods with the information that it is of course a conversion that not every array supports. (Arrays are not necessarily contiguous or in order.) I think that this information needs to be stated again, since it is non-obvious to the users that start out.